src/presenters/admin/indexation-permalink-warning-presenter.php

```diff
@@ -51,6 +56,10 @@ protected function get_text_for_reason( $reason ) {
 				$text = \esc_html__( 'Because of a change in your category URL setting, some of your SEO data need to be reprocessed.', 'wordpress-seo' );
 				break;
 
+			case static::REASON_HOME_URL_OPTION:
+				$text = \esc_html__( 'Because of a change in your home URL option, some of your SEO data need to be reprocessed.', 'wordpress-seo' );
```

Comment:
I'd keep the text consistent and refer to 'setting' over 'option'.

Reply:
right